agent/util-scripts/pbench-register-tool

```diff
@@ -88,13 +88,23 @@ pbench_bin="$(realpath -e ${script_path}/..)"
 #-
 
 # Defaults
-remotes_arg="localhost"
+name=""
+remotes_arg=""
+remotes_def="localhost"
+labels_arg=""
 group="default"
+tool_type="transient"
 options=""
 do_install=1
-_do_test_labels=0
+_do_test_args=0
 
 function usage() {
+	if [[ ${_do_test_args} -eq 1 ]]; then
+		# We emit a usage message now if we are not working on behalf
+		# of pbench-register-tool-set, since it will handle its own
+		# usage message on failure.
```

Comment:
In this context, the message should say something more like
```
		# Since we are working on behalf of pbench-register-tool-set,
		# skip printing the usage message here -- it will issue its own
		# usage message on failure.
```

Reply:
Done